SIS_import/sis_upload.rb

```diff
@@ -503,14 +571,19 @@ def create_instructor_sandbox_site(zip_file_name, outputFile)
 # read the settings from properties files
 upload_error = get_settings(securityFile, propertiesFile)
 
-outputFile = false
-if (!upload_error)
-	#open the output file
-	begin
-		# get output file name
-		output_file_base_name = "Canvas_upload_" + Time.new.strftime("%Y%m%d%H%M%S")
-		outputFile = File.open($outputDirectory + output_file_base_name + ".txt", "w")
+#open the output file
+begin
+	# get output file name
+	output_file_base_name = "Canvas_upload_" + Time.new.strftime("%Y%m%d%H%M%S")
+	output_file_name = $outputDirectory + output_file_base_name + ".txt"
+	outputFile = File.open(output_file_name, "w")
+	$logger.info "log file is at #{output_file_name}"
```

Comment:
There is no previous/old log file. The logger defaults to STOUT at first. Upon the success of opening this log file, logger  writes into this file then.